src/include/txn_inline.h

```diff
@@ -1132,14 +1119,18 @@ __wt_txn_read(
             if (cbt->upd_value->skip_buf) {
                 cbt->upd_value->buf.data = NULL;
                 cbt->upd_value->buf.size = 0;
-            }
+            } else if (is_ovfl_rm)
+                /*
+                 * We race with checkpoint reconciliation removing the overflown items. Retry the
```

Comment:
Can we explain why it is not ok here?

Reply:
The base value is the one we apply modify on. It should be a valid value and cannot be overflown removed.